docs/rules/custom-event-name-casing.md

```diff
@@ -49,7 +49,16 @@ export default {
 
 ## :wrench: Options
 
-Nothing.
+
+```json
+{
+  "vue/custom-event-name-casing": ["error", {
+    "ignores": []
+  }]
+}
+```
+- `ignores` (`string[]`) ... The event names to ignore. Sets the event name to allow. For example, custom event names, Vue components event with special name, or Vue library component event name. You can set the regexp by writing it like `"/^name/"` or `update:input` or `fooBar`.
```

Comment:
`update:*` is already ignored without using the `ignores` option, so I think it's easier for people to understand by using the `click:row` in Vuetify.

```suggestion
- `ignores` (`string[]`) ... The event names to ignore. Sets the event name to allow. For example, custom event names, Vue components event with special name, or Vue library component event name. You can set the regexp by writing it like `"/^name/"` or `click:row` or `fooBar`.
```

Reply:
This is accidentally the same I swear 😂